ceph-docker-build/config/definitions/ceph-docker-build.yml

```diff
@@ -0,0 +1,46 @@
+- job:
+    name: 'ceph-docker-build'
+    node: 'centos7 && x86_64 && small && !sepia'
+    concurrent: true
+    defaults: global
+    display-name: 'ceph-docker: image builds'
+    quiet-period: 5
+    block-downstream: false
+    block-upstream: false
+    retry-count: 3
+    properties:
+      - github:
+          url: https://github.com/ceph/ceph-docker
+    logrotate:
+      daysToKeep: 15
+      numToKeep: 30
+
+    triggers:
+      - github
+
+    parameters:
+      - string:
+          name: CEPH_RELEASE
```

Comment:
I've amended and re-pushed as a parameterized job now, so we build the jewel, kraken, and luminous containers.